Purchases/Public/RCEntitlementInfo.m

```diff
@@ -3,6 +3,8 @@
 //  Copyright © 2019 RevenueCat. All rights reserved.
 //
 
+@import PurchasesCoreSwift;
```

Comment:
would we eventually not need the `#import "RCEntitlementInfo.h"` ? if you have a favorite resource describing swift/obj c enum compatibility, let me know 😅

Reply:
That's exactly right, eventually all will be in swift, and we won't need any `@import` inside our SDK. This is just temporary while we move it all over.

Reply:
yeah, once we have everything in a single swift module, we don't need an import at all - everything within the same Swift module is auto-imported as part of the same scope (assuming the privacy is `internal` or `public`)

Reply:
as for resources, Apple has a few pages that are pretty thorough (but sadly they don't go very deep into migrating frameworks): 
https://developer.apple.com/documentation/swift/migrating_your_objective-c_code_to_swift
https://developer.apple.com/documentation/swift/imported_c_and_objective-c_apis

Reply:
😂 and I just realized I was too slow for this one